### PR TITLE
profile can be edited

### PR DESCRIPTION
**Pull request by miku0129**

### CODE REVIEW

File: client/src/components/Profile/Profile.jsx [hunk 1]
```diff
@@ -6,29 +6,52 @@ import { faUserCircle } from "@fortawesome/free-solid-svg-icons";
...
+        "email":email,
+        "phone":phone
+      }); 
+
+
+      //profile page is updated exept email. 
+      //email shouldn't be changed 
+      user.attributes.given_name=givenName; 
+      user.attributes.family_name=familyName;
+      user.attributes.phone_number=phone;
```
Comment (catboop) [inline client/src/components/Profile/Profile.jsx]: `user` cannot be changed this way. Please remove. We will need to fix this bug later!

File: client/src/components/Profile/Profile.jsx [hunk 2]
```diff
@@ -90,24 +113,25 @@ function Profile() {
...
                 )}
                 {displayInputs && (
                   <div className="attribute">
-                    <input
+                    {/* <input
                       className="profile-input"
                       value={email}
                       onChange={(e) => setEmail(e.target.value)}
-                    />
+                    /> */}
```
Comment (catboop) [inline client/src/components/Profile/Profile.jsx]: just delete

File: client/src/components/Profile/Profile_bk [hunk 3]
```diff
@@ -0,0 +1,164 @@
+import React, { useState, useEffect, useContext } from "react";
```
Comment (catboop) [inline client/src/components/Profile/Profile_bk]: please delete file

**Review by catboop — CHANGES_REQUESTED** [review]

File: client/src/components/Profile/Profile.jsx (hunk 2 above)
Comment (miku0129) [reply client/src/components/Profile/Profile.jsx]: thx I did it!

File: client/src/components/Profile/Profile_bk (hunk 3 above)
Comment (miku0129) [reply client/src/components/Profile/Profile_bk]: thx I did it!

File: client/src/components/Profile/Profile.jsx (hunk 1 above)
Comment (miku0129) [reply client/src/components/Profile/Profile.jsx]: Thx!

**Review by catboop — APPROVED** [review]

**Pull request merged**